### ISSUE TITLE
Feature: query history

### ISSUE THREAD

**Issue by rrbarbosa** [sqlectron/sqlectron-gui#277]
It would be nice if we could store our query history per server. Maybe backed up by a sqlite DB, or something similar. If we agree on the basic functionality, I could help with development.

**Comment by maxcnunes** [comment]
The app already has initial support for that. Every executed query is stored in memory. I had implemented that thinking in finishing this in the future. But I never got the time to finish that.
Here is the code where the query is stored after been executed: https://github.com/sqlectron/sqlectron-gui/blob/master/src/renderer/reducers/queries.js#L60
I thought it could have like a select element that would allow select any previous query executed for that tab in the current session. Is that what you were thinking?

**Comment by rrbarbosa** [comment]
No, ideally I would like to keep history per server, so they are persisted over sessions. Other clients, like Postico or Sequel Pro have a button on the UI that allows you to view and search past queries, a little like a bash history. What do you think?

I think if we keep the size of the history small, ~100 queries, it should be fine to keep them in plain text.

**Comment by maxcnunes** [comment]
I think that button would be even better if the user could filter by past queries for only the current tab as well. So the user have access to all queries in that session. But if he want to see only the queries for that current tab he is able to filter that. Could be a different button or just a check `show only queries of current tab`.

**Comment by jonatascb** [comment]
I would like to support that feature as well. I think that would be very useful.

**Comment by AgDude** [comment]
psql for postgres has similar functionality where you can use ctrl-r and start typing just like in bash. To me that would be the ideal functionality, but a UI to view and search history would be nice too.
I also would consider contributing to this.
